lib/utils/CrossfilterHistogramPlot.ts

```diff
@@ -1,4 +1,5 @@
 import * as d3 from "../deps/d3.ts";
+import { effect, signal } from "@preact/signals-core";
```

Comment:
Should get a linting rule for this (or use biome to sort), but I personally prefer having third-party imports at the top, followed by relative imports.